hooks/post_gen_project.py

```diff
@@ -9,13 +9,31 @@ def remove_publish_pypi_github_action():
     os.remove(".github/workflows/python-publish.yaml")
 
 
+def generate_docker_file(yes, python_version):
+    if not yes:
+        os.remove("Dockerfile_template")
+        return
+    with open("Dockerfile_template", "r") as rf:
+        docker_file_content = rf.read().format(PYTHON_VERSION=python_version)
+        rf.close()
+    with open("Dockerfile", "w") as wf:
+        wf.write(docker_file_content)
+        wf.close()
+    os.remove("Dockerfile_template")
```

Comment:
Why not use cookiecutter context directly and name the file as Dockerfile?

Reply:
Umm.... you mean cookiecutter context could be used directly even in a non-python file?

Reply:
Yep, that's what I mean